docs/ruleset_functions/is_zone_mechanically_cooled.md

```diff
@@ -0,0 +1,45 @@
+# is_zone_mechanically_cooled
+**Schema Version:** 0.0.25  
+
+**Description:** determines whether a zone is cooled.  Checks for transfer air
+
+**Inputs:**
+- **RMI**
+- **zone**
+
+**Returns:**  
+- **has_cooling_system**: a boolean - TRUE or FALSE
+ 
+**Function Call:**
+- **get_list_hvac_systems_associated_with_zone**
+
+## Logic:
+- set a boolean has_cooling_system to false: `has_cooling_system = FALSE`
+- get the hvac systems serving the zone by using the get_list_hvac_systems_associated_with_zone function: `list_hvac_systems = get_list_hvac_systems_associated_with_zone(RMI,zone)`
+	- loop through each HeatingVentilatingAirconditionsSystem system: `for system in list_hvac_systems:`
+		- look for a cooling system: `if system.cooling_system != NULL or system.cooling_system != NONE:`
+			- there is a cooling system: `has_cooling_system = TRUE`
```

Comment:
@mkarpman and @KarenWGard in addition to verifying that the cooling system is not equal tto NONE o NULL, should we be verifying if the cooling capacity >= 3.4 Btu/(h*ft2))?

Reply:
discussed on call 230615 - @mkarpman indicates that the language states "cooling system" and not specifically a cooling capacity.